pkg/query-service/app/logs/v4/query_builder.go

```diff
@@ -149,7 +149,7 @@ func buildAttributeFilter(item v3.FilterItem) (string, error) {
 			return fmt.Sprintf(logsOp, keyName, fmtVal), nil
 		case v3.FilterOperatorContains, v3.FilterOperatorNotContains:
 			// we also want to treat %, _ as literals for contains
-			val := utils.QuoteEscapedStringForContains(fmt.Sprintf("%s", item.Value))
+			val := utils.QuoteEscapedStringForContains(fmt.Sprintf("%s", item.Value), false)
```

Comment:
Why is this false? Why can't the same apply here?

Reply:
Not required as if we store it in the attributes we store it is stored as

```
'key1.2':'["a","b","c"]'
````

While in the resources labels it is stored as ` "[\"a\",\"b\",\"c\"]"`



![image](https://github.com/user-attachments/assets/4b162686-3a98-4cbe-90e2-ba8aa8fb8704)

![image](https://github.com/user-attachments/assets/05c7a2d3-888a-4c64-b291-0a1f8d7c5f28)